docs/user-guide/README.md

```diff
@@ -64,6 +64,9 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
 [**Overview**](overview.md)
 : A brief overview of Kubernetes concepts.
 
+**Cluster**
+: A cluster is the compute resources on top of which your containers are built.
```

Comment:
I see. I add this because I saw that Cluster was mentioned several times in some of the documents (such as [Kubernetes 101](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/walkthrough/README.md)) without explaining it. I think that is likely to confuse first-time users.